### PR TITLE
feat(icons): add Relay EmptyStates and flag icon

### PR DESCRIPTION
**Pull request by frogzsj**
IconRelayFlag:
image

EmptyStates:
image

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @frogzsj has signed the CLA. Thanks for the pull request!

File: src/icons/general/IconRelayFlag.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// @flow
+import * as React from 'react';
+
+import { bdlGray50 } from '../../styles/variables';
+import AccessibleSVG from '../accessible-svg';
+
+import type { Icon } from '../flowTypes';
+
+const IconWorkflow = ({ className = '', height = 16, title, width = 16, color = bdlGray50 }: Icon) => (
```
Comment (tjgupta) [inline src/icons/general/IconRelayFlag.js]: Should this be `IconRelayFlag`?

Comment (frogzsj) [reply src/icons/general/IconRelayFlag.js]: 🤦‍♂

File: src/icons/states/RelayLiteEmptyState.js [hunk 2]
```diff
@@ -0,0 +1,352 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg/AccessibleSVG';
+import { bdlBoxBlue, bdlBoxBlue20 } from '../../styles/variables';
+
+import type { Icon } from '../flowTypes';
+
+const RelayLiteEmptyState = ({ className = '', height = 140, title, width = 140 }: Icon) => (
```
Comment (priyajeet) [inline src/icons/states/RelayLiteEmptyState.js]: Can this file be cleaned up/re exported? Lots of transforms and g tags that are likely unnecessary

File: src/icons/states/RelayFullEmptyState.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg/AccessibleSVG';
+import { bdlBoxBlue, bdlBoxBlue10 } from '../../styles/variables';
+
+import type { Icon } from '../flowTypes';
+
+const RelayFullEmptyState = ({ className = '', height = 140, title, width = 140 }: Icon) => (
+    <AccessibleSVG
```
Comment (priyajeet) [inline src/icons/states/RelayFullEmptyState.js]: If you haven't already, I would run this through svg optimizor to see if it can become smaller

**Pull request closed without merging**